code/_onclick/adjacent.dm

```diff
@@ -25,50 +25,42 @@
 	* If you are diagonally adjacent, ensure you can pass through at least one of the mutually adjacent square.
 		* Passing through in this case ignores anything with the throwpass flag, such as tables, racks, and morgue trays.
 */
-/turf/Adjacent(var/atom/neighbor, var/atom/target = null)
+/turf/Adjacent(atom/neighbor, atom/target = null, atom/movable/mover = null)
```

Comment:
they are null by default

Reply:
Yea, that's copied from /tg/ but I can clean it.